calendar-bundle/src/Resources/contao/modules/ModuleEventlist.php

```diff
@@ -99,6 +100,11 @@ protected function compile()
 		$intMonth = Input::get('month');
 		$intDay = Input::get('day');
 
+		if (\is_array($intYear) || \is_array($intMonth) || \is_array($intDay))
```

Comment:
I’d cast the values instead I think.

```php
$intYear = (int) Input::get('year');
$intMonth = (int) Input::get('month');
$intDay = (int) Input::get('day');
```

Reply:
Interesting. Casting to string gives a `Warning: Array to string conversion` but casting to int seems to be no problem. 😄

Reply:
Changed in c8995f1152e9e9b84ec3a0c9259427cfa0316c27. Hope PHP does not add a `Warning: Array to int conversion` anytime soon. 😄